packages/core/src/renderer/components/dock/logs/logs-view-model.injectable.ts

```diff
@@ -28,7 +28,7 @@ export interface InstantiateArgs {
 const logsViewModelInjectable = getInjectable({
   id: "logs-view-model",
 
-  instantiate: (di, { tabId }: InstantiateArgs) => new LogTabViewModel(tabId, {
+  instantiate: async (di, tabId: TabId) => new LogTabViewModel(tabId, {
```

Comment:
nit: it shouldn't be necessary to specify the type of `tabId` here as well, specifying it only in the use of `lifecycleEnum.keyedSingleton` should be enough